libs/testing/src/user.session.ts

```diff
@@ -331,4 +333,17 @@ export class UserSession {
       await this.testAgent.post(`/v1/changes/${change._id}/apply`);
     }
   }
+
+  async createSubscriber(fields: Partial<SubscriberEntity> = {}) {
```

Comment:
We have the SubscriberService to create subscribers in testing. Does it not adapt to your needs testing here?